Sources/StreamChat/Query/Filter+ChatChannel.swift

```diff
@@ -180,15 +180,15 @@ extension Filter where Scope == ChannelListFilterScope {
                 return nil
             }
             return NSPredicate(
-                format: "%K BEGINSWITH[c] %@",
+                format: "%K BEGINSWITH[c] %@".prepend(("ANY "), ifCondition: isACollectionFilter),
                 argumentArray: [keyPathString, prefix]
             )
         case .contains where mappedValue is String:
             guard let needle = mappedValue as? String else {
                 return nil
             }
             return NSPredicate(
-                format: "%K CONTAINS %@",
+                format: "%K CONTAINS %@".prepend(("ANY "), ifCondition: isACollectionFilter),
```

Comment:
Are we sure we need this for contains? 🤔 I don't think so. Can we unit test this?

Reply:
This is actually needed for the cases like this one `.contains(.memberName, value: "test")`. I added the related testCase.